playground-common/src/PSGenerator/Common.hs

```diff
@@ -51,6 +51,9 @@ psJsonEither =
 psJsonMap :: MonadReader BridgeData m => m PSType
 psJsonMap = TypeInfo "" "Data.Json.JsonMap" "JsonMap" <$> psTypeParameters
```

Comment:
Is `psJSONMap` still needed now?

Reply:
Ah no. Neither's `psJsonEither`. I'll remove them. Thanks. :-)